modules/integration/tests-common/integration-tests-utils/src/main/java/org/wso2/mb/integration/common/utils/ui/pages/main/QueueSubscriptionsPage.java

```diff
@@ -43,6 +49,81 @@ protected QueueSubscriptionsPage(WebDriver driver) {
     }
 
     /**
+     * Search queue subscriptions according to the search criteria.
+     *
+     * @param queueNamePattern string pattern of the queue name (* for all)
+     * @param identifierPattern string pattern of the identifier (* for all)
+     * @param ownNodeIdIndex index of the node Id in the dropdown the subscriptions belong to
+     * @return number of subscriptions listed under search result
+     */
+    public int searchQueueSubscriptions(String queueNamePattern, String identifierPattern, int ownNodeIdIndex,
+                                        boolean isNameExactMatch, boolean isIdentifierExactMatch) {
+
+        WebElement queueNamePatternField = driver.findElement(By.name(UIElementMapper.getInstance()
+                .getElement("mb.search.queue.name.pattern.tag.name")));
```

Comment:
This is not just a string. These are from mapper.properties file. So no need of making constants out of this.